avm/ptn/authorization/policy-exemption/tests/e2e/mg.max/main.test.bicep

```diff
@@ -0,0 +1,78 @@
+targetScope = 'managementGroup'
+metadata name = 'Policy Exemptions (Management Group scope)'
+metadata description = 'This module deploys a Policy Exemption at a Management Group scope using common parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'apemgmax'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Required. The policy definition ID to assign the policy to.')
+param policyDefinitionID string = '/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d'
```

Comment:
Same comment as in min file

Reply:
I'll not comment this in every file now, so either please respond in one of the 2 comments, or if you agree with the suggestion, please implement it for the other test files too :)

Reply:
Think I'm going to keep as is to avoid breaking this linter rule: ![alt text](image.png) as I think it'll be confusing for people who are using custom policies. Open to discussing though.

Reply:
Feel free to reopen if you want to discuss more.

Reply:
Hey @oZakari,
unfortunately I would 😄 But it's not a big one.

The warning you're avoidining by moving the reference to a parameter is not exactly 'addessed' doing so 😄 What Bicep wants you to do is to add a reference 

```bicep
resource policyDefinition 'Microsoft.Authorization/policyDefinitions@2025-01-01' existing = {
  name: '06a78e20-9358-41c9-923c-fb736d382a4d'
  scope: tenant()
}

# Example ref
output test string = policyDefinition.id
```

to the file and reference its ID instead.

The above example would return `/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d`.

Personally I'm not the biggest fan of that Linter rule (or at least that it applies for every ID reference), but it also has its merits.

Reply:
Ah that makes more sense. Appreciate the explanation. I have resolved the issue. Let me know if you need anything else, as hoping to get this closed out this week if possible ☺️